src/Compilers/CSharp/Portable/BoundTree/Expression.cs

```diff
@@ -141,7 +142,7 @@ private static IArgument DeriveArgument(int parameterIndex, int argumentIndex, I
                 }
 
                 // There is no supplied argument and there is no params parameter. Any action is suspect at this point.
-                return new SimpleArgument(null, null);
+                return new SimpleArgument(null, new InvalidExpression(invocationSyntax));
```

Comment:
I had a sinking feeling when I wrote the line above that something like this might turn out to be necessary. Sorry for not taking care of it then.